src/GameSelector/UserGame.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, HStack, Text, VStack } from 'native-base';
+import React from 'react';
+import { Pressable } from 'react-native';
+import { Board } from '../GameState/Board';
+import { Game } from '../interface';
+
+interface UserGameProps {
+  navigateToGameState: (id: number | undefined) => void;
+  game: Game | undefined;
+}
+
+const GRADIENT_COLORS: any = [];
+GRADIENT_COLORS[0] = {
```

Comment:
You should give domain details to your constants. Here it's not a gradient, but the colour of an in-progress game, or a successful game.